client/simple/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "simple",
+  "version": "1.0.0",
+  "type": "module",
+  "scripts": {
+    "clean": "rm -Rf node_modules",
+    "build": "node theme_icons.js && vite build",
+    "fix": "eslint --fix && stylelint --fix strict 'src/**/*.{css,scss,sass,less,styl,vue,svelte}'",
+    "icons.html": "node theme_icons.js"
+  },
+  "devDependencies": {
+    "@eslint/js": "^9.19.0",
+    "copy-webpack-plugin": "^12.0.2",
+    "css-loader": "^7.1.2",
+    "edge.js": "^6.2.1",
+    "eslint": "^9.19.0",
+    "filemanager-webpack-plugin": "^8.0.0",
+    "globals": "^15.14.0",
+    "ionicons": "^7.4.0",
+    "leaflet": "^1.9.4",
+    "less": "^4.2.2",
+    "less-loader": "^12.2.0",
+    "normalize.css": "^8.0.1",
+    "sharp": "^0.33.5",
+    "style-loader": "^4.0.0",
+    "stylelint": "^16.14.1",
+    "stylelint-config-standard": "^37.0.0",
+    "stylelint-config-standard-less": "^3.0.1",
+    "stylelint-prettier": "^5.0.3",
+    "svgo": "^3.3.2",
+    "vite": "^6.0.7",
+    "vite-plugin-static-copy": "^2.2.0",
+    "vite-plugin-stylelint": "^6.0.0",
+    "webpack": "^5.97.1",
+    "webpack-cli": "^6.0.1"
+  },
+  "dependencies": {
+    "autocomplete-js": "^2.7.1"
```

Comment:
This dependency is unmaintained since 6 years, we shouldn't use this anymore (not related to this PR) ...

I'll make a PR to replace it in some time...

Reply:
Thats true .. we have such old unmaintained libs .. 

> I'll make a PR to replace it in some time...

:rocket: .. and we should take in mind there are open tasks --> https://github.com/searxng/searxng/issues?q=label%3A%22area%3A%20autocomplete%22%20state%3Aopen

But first lets migrate the code "as is" into the new environment for our FE development (vite), than we can migrate the JS code base to JS modules .. and when we have JS modules we can implement a new autocompleter and improve other client stuff.

Reply:
That warning was too late, already implemented in https://github.com/searxng/searxng/pull/4318.

I can rebase it after this PR is merged though.

Reply:
> I can rebase it after this PR is merged though.

I will see what will be easier .. but not today :-)